index.html

```diff
@@ -11,6 +11,13 @@
 <body>
     <main>
         <header>
+            <nav>
+                <ul>
+                    <li>Home</li>
+                    <li>ABOUTs</li>
```

Comment:
może będzie bardziej spójnie jak napiszemy Abouts

Reply:
ok to ma sens